src/booster.jl

```diff
@@ -441,6 +526,13 @@ followed by [`update!`](@ref) for `nrounds`.
 `watchlist` is a dict the keys of which are strings giving the name of the data to watch
 and the values of which are [`DMatrix`](@ref) objects containing the data.
 
+`early_stopping_rounds` if 0, the early stopping function is not triggered. If set to a positive integer, 
+training with a validation set will stop if the performance doesn't improve for k rounds.
+
+`maximize` If early_stopping_rounds is set, then this parameter must be set as well.
```

Comment:
This seems to imply that the user must input the value whenever they input `early_stopping_rounds`.  Maybe better to say something like "only used if `early_stopping_rounds > 0`".